python-avd/pyavd/_eos_designs/structured_config/network_services/dps_interfaces.py

```diff
@@ -37,8 +37,11 @@ def dps_interfaces(self: AvdStructuredConfigNetworkServicesProtocol) -> None:
         # TODO: do IPv6 when needed - for now no easy way in AVD to detect if this is needed
         # When needed - need a default value if different than IPv4
 
-        dps1.flow_tracker = self.shared_utils.get_flow_tracker(
-            self.inputs.fabric_flow_tracking.dps_interfaces, output_type=EosCliConfigGen.DpsInterfacesItem.FlowTracker
+        # Using _update to allow Undefined to be set.
+        dps1._update(
+            flow_tracker=self.shared_utils.get_flow_tracker(
+                self.inputs.fabric_flow_tracking.dps_interfaces, output_type=EosCliConfigGen.DpsInterfacesItem.FlowTracker
+            )
         )
```

Comment:
Would this not make more sense rather than shoving a potentially Undefined?
```suggestion
        if (flow_tracker := self.shared_utils.get_flow_tracker(
                self.inputs.fabric_flow_tracking.dps_interfaces, output_type=EosCliConfigGen.DpsInterfacesItem.FlowTracker)):
            dps1.flow_tracker = flow_tracker
```

Reply:
The undefined is already there by the time we assign it. So it makes no difference and this is simpler (and same as in other places)